src/main/java/io/github/mivek/model/WeatherCondition.java

```diff
@@ -89,10 +95,14 @@ public void addPhenomenon(final Phenomenon pPhenomenon) {
      * @return true if there is at least phenomenon.
      */
     public boolean isValid() {
-        return !phenomenons.isEmpty() || (intensity != null && descriptive != null);
+        if (Descriptive.SHOWERS == descriptive) {

```

Comment:
Sorry, I meant Thunderstorm I my last comment.
I think it can be simplier

`return !phenomenons.isEmpty() || Descriptive.Thunderstorm == descriptive`

After this change, the PR is ok for me

Reply:
done!